course2/meeting10/main.cc

```diff
@@ -0,0 +1,54 @@
+#include <random>
+#include <algorithm>
+#include <vector>
+#include <list>
+#include <deque>
+#include <iostream>
+#include <chrono>
+#include <functional>
+
+struct RNG
+{
+	std::default_random_engine randEngine;
+	std::uniform_real_distribution<double> realDist{0, 1000};
+	double operator() () { return realDist(randEngine); }
+};
+
+void PrintTime(std::function<void()> func, const char* message)
```

Comment:
Github erased the template. I meant `std::function<void()>` and `std::function<int(double,double)>`